scalr/data_source_environment_test.go

```diff
@@ -11,6 +12,8 @@ import (
 func TestAccEnvironmentDataSource_basic(t *testing.T) {
 	rInt := GetRandomInteger()
 
+	cuttedRInt := strconv.Itoa(rInt)[:len(strconv.Itoa(rInt))-2]
```

Comment:
`GetRandomInteger` can return anything, even in range 0:99, so trick with `[:len(strconv.Itoa(rInt))-2]` in above case will not work. Try this
```
do {rInt := GetRandomInteger()} while (rInt < 1000)
```